### ISSUE TITLE
Broken conversion from negative number to string

### ISSUE THREAD

**Issue by Lartu** [Lartu/ldpl#79]
When a negative number (for example `-12.2`) is stored in a TEXT variable, a 0 is appended before it.

For example this:
```coffeescript
data:
strVar is text

procedure:
store -12.989 in strVar
display strVar crlf
```

Displays `0-12.989`.

**Comment by dgarroDC** [comment]
It's working fine for me, it displays `-12.989`. I think is was just a bug in the conflow test, I fixed it in Lartu/ldpltest#2.

**Comment by Lartu** [comment]
:man_facepalming:  Oooof.

Thank you very very much, haha!
